Artifacts.toml

```diff
@@ -0,0 +1,255 @@
+["pd_nc_sr_pbesol_standard_0.4.1_psp8"]
+git-tree-sha1 = "d5d914c4b8ae3119fae60f3a3e66225db41de579"
+lazy = true
+
+    [["pd_nc_sr_pbesol_standard_0.4.1_psp8".download]]
+    sha256 = "6870faeceb4f68196901160a6ac455016dc1b9b1ea4ec629607d8e1757d9055f"
+    url = "https://github.com/JuliaMolSim/PseudoLibrary/raw/main/artifacts/pd_nc_sr_pbesol_standard_0.4.1_psp8.tar.gz"
```

Comment:
Is it a good idea to point these to main (i.e. a potentially moving branch)? Might be better to use the exact commit hash of the PseudoLibrary repository. Also I've noticed other projects use a different API to access raw data, e.g. in this case:

https://raw.github.com/JuliaMolSim/PseudoLibrary/d37ddfb0227c5d071ab27228340a0cd78b401719/artifacts/pd_nc_sr_pbesol_standard_0.4.1_psp8.tar.gz

Reply:
BTW also relevant to your most recent comment.

Reply:
😅 I just committed the update to the scripts and removed the pseudos, using another slightly different URL format.

I think we can standardize on what you suggest or on what's used now in the Artifacts TOML (this is generated by a script in the PseudoLibrary, if we want to change the URL format, it would be best to update the script there).

Reply:
BTW you missed some pseudos. Not all are removed yet.

Reply:
Ah oops, there are still some in the QE examples. or is it the Abinit examples that I need to go through too?